index.html

```diff
@@ -66,10 +66,11 @@ <h1>New Entry</h1>
           <h1 class="inline">Entries</h1>
         </div>
         <div class="column-half-entries text-right">
-          <button class="new-button tab">NEW</button>
+          <button class="new-button" data-view="entries">NEW</button>
```

Comment:
This button shows the entry form, so it should be `data-view="entry-form"`. Also, it should be an `a` element, it is responsible for "navigation" between views.